formPage.php

```diff
@@ -58,4 +64,4 @@
     </div>
```

Comment:
Does this Div closer actually close anything? I've looked at it a couple of times and can't see what it's for.

Reply:
Checked the div tags, there's no strays in there.